srcpkgs/wmix/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'wmix'
+pkgname=wmix
+version=3.5
+revision=1
+build_style=gnu-configure
+makedepends="alsa-lib-devel libXpm-devel libXrandr-devel"
+short_desc="Dockapp to control ALSA sound. Channel selector and keyboard support"
```

Comment:
I think better `short_desc` is "WindowMaker's dockapp mixer for OSS or ALSA"

Reply:
Will make those changes and push shortly.